natgeo/mortar
# [SYTO-729] Build video playlist component in Web Tools/Mortar (Elevate AEM's component)

This PR is intended to fix some problems in the video playlist component
https://jira.natgeo.com/browse/SYTO-729

How to test

1. ``npm i``
2. ``gulp serve``
3.Go to http://localhost:3000/components/
4. Check the videoplaylist component
5. Click on the first playlist's video with autocontinue true and be sure that video is playing.
6. Check all functionality with the playlist with autocontinue false.
